docs/pages/link.js

```diff
@@ -233,14 +233,16 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
 `}
           />
         </MainSection.Subsection>
-        {/* <MainSection.Subsection columns={2}>
+        <MainSection.Subsection columns={2}>
           <MainSection.Card
             type="do"
-            description={`\`isExternal\` PROP COMING SOON.
-
-Use the "visit" icon to represent an external Link/domain in a text context. The icon should match the text size. Please note: Disclaimers and links taking users to Pinterest sub-sites don't need to display an external link icon.`}
+            description={`
+Use the "visit" icon to represent an external Link/domain in a text context. The icon should match the text size and color. Please note: Disclaimers and links taking users to Pinterest sub-sites don't need to display an external link icon.`}
```

Comment:
I'm not sure I understand the word "disclaimers" in this context?

Reply:
Removed as per conversation with @cintiapub 

Disclaimers are texts like “By continuing, you agree to Pinterest’s Business Terms of Service…”